### DISCUSSION TITLE
Local development using Planetscale / Planetscale drivers?

### DISCUSSION THREAD

**Discussion by Fr3akyMurk** [planetscale/discussion#616] [Q&A] [answered]
Greetings.
I want to use a specific template to try out locally without much mixing and mashing (i tried already, it went horribly wrong) and so now i am wondering if there is any way to use the same set up but somehow convert it to local db, so basically bring your own mysql server.

I use @planetscale/database and @drizzle-orm/planetscale-serverless.
And this is my configuration file for drizzle.

```javascript
import { Client } from "@planetscale/database";
import { drizzle } from "drizzle-orm/planetscale-serverless";

import { env } from "@/env";
import * as schema from "./schema";

const connection = new Client({
    url: "localhost:3306",
    username: "MariaInterface",
    password: "MARIADB"
  });

export const db = drizzle(connection, { schema });
```

When i use planetscales own link i get a bunch of ssl errors that varies from SSL_INCORRECT_VERSION_NUMBER to HANDSHAKE_FAILED_NO_SSL_SUPPORT (This is not the exact correct error log but this is what i can remember)

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @Fr3akyMurk,

Since `@planetscale/database` connects to your database over HTTP, you can't connect it directly to a locally running MySQL.

Please give: https://github.com/mattrobenolt/ps-http-sim a try. This sits in front of a local MySQL and makes it `@planetscale/database` compatible.
